### DISCUSSION TITLE
Close tab on the left

### DISCUSSION THREAD

**Discussion by Willdrick** [rafaelmardojai/firefox-gnome-theme#298] [Q&A]
I'm trying to have the tab close button on the left side, any idea how to do it? 
I found a `Put tab close button and icon sound to the right` on `/chrome/firefox-gnome-theme/theme/parts/tabsbar.css` (line 167), but I have no idea what I should modify (or even _if_ I should modify)

**Comment by victormihalache** [comment]
I had a similar suggestion for WhiteSur GTK, check this thread to see how they do it now: https://github.com/vinceliuice/WhiteSur-gtk-theme/issues?q=close
